pkgs/misc/lz4json/default.nix

```
@@ -0,0 +1,38 @@
+{ pkgs ? import <nixpkgs> { system = builtins.currentSystem; }
+, lib ? pkgs.lib
+, stdenv ? pkgs.stdenv
+, fetchFromGitHub ? pkgs.fetchFromGitHub
+}:
+
+let
+  name = "lz4json";
+  version = "c44c51005c505de2636cc1e59cde764490de7632";  
+  sha256 = "sha256-rLjJ7qy7Tx0htW1VxrfCCqVbC6jNCr9H2vdDAfosxCA=";
+in
+(stdenv.mkDerivation {
+
+  name="${name}";
+  src = fetchFromGitHub {
+    owner = "andikleen";
+    repo = "lz4json";
+    rev = "${version}";  
+    sha256 = "${sha256}";  
+  };
+
+  buildInputs = [ pkgs.lz4  pkgs.pkg-config ];
+
+  buildPhase = ''
+    mkdir -p $out/bin
+    make 
+    install -D lz4jsoncat $out/bin/lz4jsoncat
+  '';
+
+  installPhase = "true";
```

```suggestion
  installPhase = ''
      mkdir -p $out/bin
      install -D lz4jsoncat $out/bin/lz4jsoncat
  '';
```

`buildPhase` should just build artifacts and use `installPhase` to install the artifacts.